app/assets/javascripts/application.js

```diff
@@ -13,3 +13,11 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+
+const foo = 2;
+let baz = 1;
+
+var x = () => 1 + 1;
```

Comment:
'arrow function syntax (=>)' is only available in ES6 (use esnext option).<br>'x' is defined but never used.